addons/fleet/migrations/12.0.0.1/pre-migration.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def keep_old_data_fleet_vehicle_model(cr):
+    openupgrade.logged_query(
+        cr, """
+            DELETE FROM ir_model_data
+            WHERE module like 'fleet'
+                  AND model like 'fleet.vehicle.model'
+                  AND name in (
+                     'model_a1',
+                     'model_a3',
+                     'model_a4',
+                     'model_a5',
+                     'model_a6',
+                     'model_a7',
+                     'model_a8',
+                     'model_agila',
+                     'model_ampera',
+                     'model_antara',
+                     'model_astra',
+                     'model_astragtc',
+                     'model_classa',
+                     'model_classb',
+                     'model_classc',
+                     'model_classcl',
+                     'model_classcls',
+                     'model_classe',
+                     'model_classgl',
+                     'model_classglk',
+                     'model_classm',
+                     'model_classr',
+                     'model_classs',
+                     'model_classslk',
+                     'model_classsls',
+                     'model_combotour',
+                     'model_corsa',
+                     'model_insignia',
+                     'model_meriva',
+                     'model_mokka',
+                     'model_q3',
+                     'model_q5',
+                     'model_q7',
+                     'model_serie1',
+                     'model_serie3',
+                     'model_serie5',
+                     'model_serie6',
+                     'model_serie7',
+                     'model_seriehybrid',
+                     'model_seriem',
+                     'model_seriex',
+                     'model_seriez4',
+                     'model_tt',
+                     'model_zafira',
+                     'model_zafiratourer'
+            )
+        """
+    )
+
+
+def keep_old_data_fleet_service_type(cr):
+    openupgrade.logged_query(
+        cr, """
+            DELETE FROM ir_model_data
+            WHERE module like 'fleet'
```

Comment:
Why are you using `like` operator instead of `=`?

Reply:
For no particular reason. I just changed it.